extensions-contrib/kafka-emitter/pom.xml

```diff
@@ -86,7 +86,6 @@
       <artifactId>slf4j-api</artifactId>
       <scope>provided</scope>
     </dependency>
-
```

Comment:
nit, can we avoid unnecessary formatting changes here since we're not making change to this file?